build.gradle

```diff
@@ -57,23 +57,33 @@ configurations {
 }
 
 dependencies {
-    implementation 'org.apache.kafka:kafka-clients:2.3.1'
-    implementation 'org.apache.kafka:kafka-streams:2.3.1'
+    implementation ('org.apache.kafka:kafka-clients:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
+    implementation ('org.apache.kafka:kafka-streams:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
     implementation 'org.apache.avro:avro:1.9.0'
```

Comment:
Should you add a entry for scala-reflect in implementation ?

Reply:
Yes, it's needed.

Reply:
Thanks!